src/utils.mjs

```diff
@@ -84,9 +82,18 @@ export const normaliseAstroOutputPath = (initialPath, options = {}) => {
     return;
   }
 
-  const normalisedPath = initialPath
-    .toLowerCase()
-    .replace(/ /g, ASTRO_PATH_JOIN_DELIMITER);
+  const [pathWithoutQueryAndFragment, queryStringAndFragment] =
+    splitPathFromQueryAndFragment(initialPath);
+
+  const pathSegments = pathWithoutQueryAndFragment.split(pathSeparator);
+
+  let normalisedPath = pathSegments
+    .map((segment) => githubSlug(segment).toLowerCase())
```

Comment:
Is there a reason why `.toLowerCase` is used here? It's not used [in astro](https://github.com/withastro/astro/blob/ad57a02c330b544770ab853fe0521eb784421016/packages/astro/src/content/utils.ts#L227). Would this cause some case problems as URLs are case sensitive

>URLs in general are case-sensitive (with the exception of machine names). There may be URLs, or parts of URLs, where case doesn't matter, but identifying these may not be easy. Users should always consider that URLs are case-sensitive. /u ([source](https://www.w3.org/TR/WD-html40-970708/htmlweb.html))

Reply:
I totally missed that 🤦 I'll remove it now and get this merged